.travis.yml

```diff
@@ -32,3 +32,11 @@ script:
   - docker-compose logs
   - curl --retry 10 --retry-delay 5 -D- http://localhost:9200/
   - curl --retry 10 --retry-delay 5 -D- http://localhost:5601/api/status
+  - docker-compose down && docker-compose rm
```

Comment:
I believe `docker-compose down` is enough, no need for `&& docker-compose rm`

Reply:
Even better, `down -v` to also wipe volumes 🗑

Reply:
Agree.

Note, I'd favor the implementation in https://github.com/deviantony/docker-elk/pull/252 over this PR. Will ask for the changes in the `.travis.yml` file in #252